oracledb-mixin/dashboards/overview.libsonnet

```diff
@@ -0,0 +1,1440 @@
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local dashboardUid = 'oracledb-overview';
+
+local prometheus = grafana.prometheus;
+local promDatasourceName = 'prometheus_datasource';
+local matcher = 'job=~"$job", instance=~"$instance"';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local lokiDatasource = {
+  uid: '$loki_datasource',
+};
+
+local databaseStatusPanel = {
+  description: 'Database status either Up or Down. Colored to be green when Up or red when Down',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [
+        {
+          options: {
+            '0': {
+              index: 0,
+              text: 'Down',
+            },
+            '1': {
+              index: 1,
+              text: 'OK',
+            },
+          },
+          type: 'value',
+        },
+      ],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'red',
+          },
+          {
+            color: 'green',
+            value: 1,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'auto',
+  },
+  pluginVersion: '9.1.8',
+  targets: [
+    {
+      datasource: promDatasource,
+      editorMode: 'code',
+      expr: 'oracledb_up{job=~"$job", instance=~"$instance"}',
```

Comment:
Expressions throughout should probably use the `matcher` variable defined above on line 8 :)